Newsletters/Yearn Finance Newsletter #48/Yearn Finance Newsletter #48.md

```diff
@@ -1,135 +1,133 @@
 
-Want to join one of the most open and transparent platforms in history? Good news: we show you exactly how you can contribute with a list of tasks just waiting for new contributors: https://contribute.yearn.rocks
+¿Quieres ser parte de una de las plataformas más abiertas y transparentes de la historia? Buenas noticias: te mostramos exactamente como colaborar con una lista de tareas esperando por nuevos contribuidores: https://contribute.yearn.rocks
 
-The future is here, it’s just not evenly distributed.
+El futuro está aquí, solo que no está se ha distribuido en partes iguales.
 
-Thank you to the Yearn community and all the contributors at Yearn who helped to make this happen. If you’d like to contribute, come join our public [discord](https://discord.gg/8rF374XkXy), create an issue in one of the [repos](https://github.com/yearn) linked in the [medium](https://medium.com/iearn/yearn-ui-v3-0-a194355bdb1f), or share your designs with us on [twitter](https://twitter.com/iearnfinance)!
+Gracias a la comunidad de Yearn y a todos los contribuidores de Yearn quienes han ayudado a hacer esto posible. Si quieres contribuir crea un Issue en uno de los [repos](https://github.com/yearn) enlazado en [medium](https://medium.com/iearn/yearn-ui-v3-0-a194355bdb1f). ¡O comparte tus diseños con nosotros en [twitter](https://twitter.com/iearnfinance)!
 
-Read an in-depth thread on Yearn’s transparency [here](https://twitter.com/iearnfinance/status/1445143482830446600?s=20).
+Lee un hilo sobre la transparencia de Yearn a profundidad [Aquí](https://twitter.com/iearnfinance/status/1445143482830446600?s=20).
 
-Check out the vaults [here](http://yearn.finance/vaults).
+Echa un vistazo a las vaults [Aquí](http://yearn.finance/vaults).
 
-## Vaults At Yearn
+## Las vaults de Yearn
 
-You can read a detailed description of the strategies for all of our active yVaults [here](https://medium.com/yearn-state-of-the-vaults/the-vaults-at-yearn-9237905ffed3).
+Puedes leer una descripción detallada de las estrategias disponibles para todas las yVaults activas [Aquí](https://medium.com/yearn-state-of-the-vaults/the-vaults-at-yearn-9237905ffed3).
 
-## Ecosystem News
+## Noticias del ecosistema
 
-[MyEtherWallet users now have access to Yearn vaults on the Android app](https://twitter.com/myetherwallet/status/1443283619867414537)
+[Los usuarios de MyEtherWallet tienen acceso a las vaults de Yearn en Android](https://twitter.com/myetherwallet/status/1443283619867414537)
 
-[storming0x has joined yAcademyDAO, where he will be starting off auditing a rebalancer strategy for Yearn](https://twitter.com/yAcademyDAO/status/1443138482604371974)
+[storming0x se ha unido a yAcademyDAO, donde realizara una auditoria de la estrategia de rebalanceo para Yearn](https://twitter.com/yAcademyDAO/status/1443138482604371974)
```

Comment:
AS-IS: [storming0x se ha unido a yAcademyDAO, donde realizara una auditoria de la estrategia de rebalanceo para Yearn](https://twitter.com/yAcademyDAO/status/1443138482604371974)

TO-BE: [storming0x se ha unido a yAcademyDAO, donde realizará una auditoria de la estrategia de rebalanceo para Yearn](https://twitter.com/yAcademyDAO/status/1443138482604371974)